examples/plotting/ImageAnimatorWCS_Example.py

```diff
@@ -0,0 +1,40 @@
+"""
+================
+ImageAnimatorWCS
+================
+
+This example shows a simple visualization for an ImageAnimatorWCS object
```

Comment:
How about:

This example shows how to create simple visualization using `ImageAnimatorWCS`